models/collectionPoint.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Event = require("./event");
+
+module.exports = (sequelize, DataTypes) => {
+  const collectionPoint = sequelize.define('collectionPoint', {
+    name: DataTypes.STRING,
+    //TODO: Add Location (coordinate)
+    createdAt: Date,
+    updatedAt: Date,
+  }, {});
+  collectionPoint.associate = function(models) {
+    // associations can be defined here
+
```

Comment:
code style: there's a lot of unnecessary empty lines here

Reply:
Ok, I'll clean up the spacing.